cfe_internal/enterprise/windows_unattended_upgrade.cf

```diff
@@ -0,0 +1,76 @@
+bundle agent windows_unattended_upgrade
+{
+  vars:
+
+    windows::
+
+      "schedule_upgrade_script"
```

Comment:
Let's use underscores to prefix these variables (trying to minimize data sent back to enterprise hub).
`_schedule_upgrade_script`, `_upgrade_script`, `_min_offset` `_tt`